app/views/curation_concerns/generic_works/_generic_work.html.erb

```diff
@@ -14,9 +14,7 @@
 
 <li id="document_<%= generic_work.to_param %>" class="chf-results-list-item document <%= render_document_class generic_work %>" itemscope itemtype="<%= generic_work.itemtype %>">
     <% doc_presenter = Chf::IndexPresenter.new(generic_work, self) %>
-
-
-    <div class="chf-results-list-item-thumb">
+    <div class="chf-results-list-item-thumb <%= "pdf_border" if doc_presenter.is_pdf? %>">
       <%# render_thumbnail_tag generic_work %>
```

Comment:
deciding whether to include `pdf_border` class in two completely different ways in two different parts of code. That may just be necessary (or really inconvenient to do different) because of how sufia is architected though? I haven't completely wrapped my head around it, but worth at least considering if this can be DRY'd in some non-awful way.